spec.html

```diff
@@ -3135,6 +3149,19 @@ <h1>SynthesizeNamedLambdaScope ( _funcExprNode_ , _enclosingScope_ )</h1>
       </emu-alg>
     </emu-clause>
 
+    <emu-clause id="sec-checkfunctionlength" aoid="CheckFunctionLength">
+      <h1>CheckFunctionLength ( _params_ , _maybeLazyNode_ )</h1>
+      <emu-alg>
+        1. Assert: _maybeLazyNode_ is a `FunctionDeclaration`, a `FunctionExpression`, a `Method`, a `Getter`, a `Setter`, or a `ArrowExpression`.
+        1. If _maybeLazyNode_ is a `Getter`, then
+            1. Let _length_ be 0.
+        1. Else,
+            1. Let _length_ be _maybeLazyNode_`.length`.
+        1. If _length_ is not equal to ExpectedArgumentCount of _params_, then
```

Comment:
Should assert that `_params_` itself is not tagged as being produced by a binary AST node.